src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,6 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
you should create three random balls



![image](https://user-images.githubusercontent.com/36365171/181482681-770c8ae9-0c7f-4f3a-b124-0fae533f1370.png)